docs/user-guide/expressions/plugins.md

```diff
@@ -249,5 +249,6 @@ That's all you need to know to get started. Take a look at this [repo](https://g
 Here is a curated (non-exhaustive) list of community implemented plugins.
 
 - [polars-business](https://github.com/MarcoGorelli/polars-business) Polars extension offering utilities for business day operations
+- [polars-distance](https://github.com/ion-elgreco/polars-distance) Polars plugin for pair-wise distance functions
```

Comment:
nitpick but I think pairwise is one word? https://www.oed.com/dictionary/pairwise_adv?tab=factsheet#32486661

Reply:
Oops :P Fixed now